lib/Viroverse/Types.pm

```diff
@@ -51,4 +55,17 @@ coerce ImporterClass,
     from StrMatch[qr/^(?!Viroverse::Import::)/],
     via { "Viroverse::Import::$_" };
 
+declare ExternalReferenceUri,
+    as Uri,
+    where {
+        my $uri = shift;
+        return $uri->abs("//")->eq($uri) &&
+            any { $uri->scheme eq $_ } qw[ http https smb ];
```

Comment:
Incidentally, a URL starting with `//` (e.g. `//example.com`) does not work, although I suspect we don't care about supporting those anyway.

Reply:
I don't think it should work; would the idea be that it infers the scheme from the context of the page content as with `<a href="//google.com">`? I'd rather require a scheme.